internal/tasks/fleet_rollout.go

```diff
@@ -79,7 +114,44 @@ func rolloutFleet(baseLog logrus.FieldLogger, fleetStore store.Fleet, devStore s
 	return nil
 }
 
-func updateDeviceSpecAccordingToFleetTemplate(ctx context.Context, log logrus.FieldLogger, devStore store.Device, orgId uuid.UUID, device *api.Device, fleet *api.Fleet) error {
+// The device's owner was changed, roll out if necessary
+func (f FleetRolloutsLogic) rolloutDevice() error {
+	f.log.Infof("Rolling out device %s/%s", f.resourceRef.OrgID, f.resourceRef.Name)
+
+	device, err := f.devStore.Get(f.ctx, f.resourceRef.OrgID, f.resourceRef.Name)
+	if err != nil {
+		return fmt.Errorf("failed to get device: %w", err)
+	}
+
+	if device.Metadata.Owner == nil || len(*device.Metadata.Owner) == 0 {
+		f.log.Info("Device has no owner")
```

Comment:
Warn or is this expected? if exected will it be over chatty at scale?

Reply:
Removed